src/PointEval/PointEvalHandler.jl

```diff
@@ -90,53 +92,88 @@ function _get_cellcoords(points::AbstractVector{Vec{dim,T}}, grid::AbstractGrid,
 end
 
 # check if point is inside a cell based on physical coordinate
-function point_in_cell(geom_interpol::Interpolation{shape}, cell_coordinates, global_coordinate) where {shape}
-    converged, x_local = find_local_coordinate(geom_interpol, cell_coordinates, global_coordinate)
-    if converged
-        return _check_isoparametric_boundaries(shape, x_local), x_local
-    else
-        return false, x_local
-    end
+# TODO linear case can be handled easier
+function point_in_cell(geom_interpol::Interpolation{shape}, cell_coordinates, global_coordinate, warn,  linesearch_max_substeps, max_iters, tol_norm) where {shape}
+    return find_local_coordinate(geom_interpol, cell_coordinates, global_coordinate, warn, linesearch_max_substeps, max_iters, tol_norm)
```

Comment:
Couldn't this function be removed? It just passes everything to `find_lcoal_coordinate`?